apps/meteor/app/models/server/raw/Sessions.ts

```diff
@@ -737,12 +739,333 @@ export const aggregates = {
 };
 
 export class SessionsRaw extends BaseRaw<ISession> {
+	protected indexes: IndexSpecification[] = [
+		{ key: { instanceId: 1, sessionId: 1, year: 1, month: 1, day: 1 } },
+		{ key: { instanceId: 1, sessionId: 1, userId: 1 } },
```

Comment:
If you change this idx to have `sessionId` as first, you won't need an individual idx on 746

Reply:
Ref: https://www.mongodb.com/docs/manual/core/index-compound/#prefixes